heartbeat.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
It seems you moved it instead of removing

Reply:
my bad :) , removed it now for sure , also removed references to private keys and mnemonic